game.js

```diff
@@ -33,6 +33,146 @@ monsterImage.onload = function () {
   monsterReady = true;
 };
 monsterImage.src = "images/monster.png";
+// Load the monster image
+var monster1Ready = false;
+var monster1Image = new Image();
+monster1Image.onload = function () {
+  // show the monster image
+  monster1Ready = true;
+};
+monster1Image.src = "images/monster.png";
+// Load the monster image
+var monster2Ready = false;
+var monster2Image = new Image();
+monster2Image.onload = function () {
+  // show the monster image
+  monster2Ready = true;
+};
+monsterImage.src = "images/monster.png";
+// Create the game objects
+var hero = {
+  speed: 256, // movement speed of hero in pixels per second
+  height: 36,
+  width: 31
+};
+var monster = {};
```

Comment:
You need two monsters here to keep track of their positions:
`var monster = {}` should become:
```
var monster1 = {}
var monster2 = {}
```